experimental/QuadFormAndIsom/src/embeddings.jl

```diff
@@ -267,17 +246,17 @@ function _subgroups_orbit_representatives_and_stabilizers(Vinq::TorQuadModuleMor
     filter!(s -> is_invariant(fV, s[2]), subs)
   end
 
-  subs = TorQuadModule[s[1] for s in subs]
-  m = gset(O, _on_subgroups, subs)
+  to_gap = get_attribute(O, :to_gap)
+  to_oscar = get_attribute(O, :to_oscar)
+
+  qgap = codomain(to_gap)
+  sgap = typeof(qgap)[sub(qgap, elem_type(qgap)[to_gap(q(lift(s[2](a)))) for a in gens(s[1])])[1] for s in subs]
```

Comment:
Better create the subgroups directly in gap and only translate representatives to Oscar. 
But in any case the number of subgroups is usually to large to put them in memory.

Reply:
I am not sure how to keep track on the form on the GAP side but yes, there are still some possible improvements to be made here. I guess that in practice, this part of the code will be called on small enough examples where we should not have too many (stable) subgroups.

Reply:
Do you need the form in the orbit/stabliser computation?

Reply:
```

    subs = collect(submodules(V; order = ord))
    filter!(s -> is_invariant(fV, s[2]), subs)
```
does not see the form

Reply:
Ah no right indeed, I forgot that the submodules are created as subgroups which inherit the form from the bigger module. So indeed yes, we could do everything on the GAP side. I can try it now and see to make a change in this PR. Thanks!

Reply:
I actually I do not know if it is better. On the GAP side, from what I see, we cannot enumerate subgroups of a given order, or stable subgroups. Which is what we use here in Hecke. I am afraid that on the GAP side we will have to many subgroups: ok we can filter them anyway but if our group has hundreds of subgroups but only 1 of the order we want, Hecke will do things better here (see https://github.com/oscar-system/Oscar.jl/issues/2835)